### ISSUE TITLE
Extra Div's Occasionally Generated

### ISSUE THREAD

**Issue by zacharybrady** [metafizzy/flickity#136]
NOTE: Could not reproduce in a reduced test case. Leaving this here though just in case this fringe case can shed light on any other issues.
![screen shot 2015-05-11 at 10 01 56 am](https://cloud.githubusercontent.com/assets/1293610/7566530/4f2202f2-f7c5-11e4-9788-d85a1618bdb8.png)
![screen shot 2015-05-11 at 10 02 10 am](https://cloud.githubusercontent.com/assets/1293610/7566533/540277fc-f7c5-11e4-9760-823277a74ee1.png)
As demonstrated in the screenshots, the .flickity-viewport and .flickity-slider container are occasionally generated twice. Which has the unfortunate result of causing this to happen to the slider:
![screen shot 2015-05-11 at 10 07 08 am](https://cloud.githubusercontent.com/assets/1293610/7566571/93affc8a-f7c5-11e4-8ba8-9536d4d6b664.png)

I don't have too many details yet other than:
- Javascript is being asynchronously loaded 
- The error never occurs when the page is refreshed
- The error ALWAYS occurs when the page is navigated to

I'm going to set up a reduced test case and also keep testing in the environment it was caught in. Will update as findings come in.

### UPDATES ###
1. Markup has no/little impact.Changed the list elements to div's and produces same results.
2. After checking a few computers and browsers I only found this issue to appear on a single computer (2013 Mac Pro) and only on the latest stable version of Chrome.

**Comment by zacharybrady** [comment]
Minor update: Markup has no/little impact.
Changed the list elements to div's and produces same results.

**Comment by desandro** [comment]
Thanks for reporting this issue. Could you provide any more context: How are you loading JS? Are you using a framework?

**Comment by zacharybrady** [comment]
I'm using Filament Group's LoadJS ( https://github.com/filamentgroup/loadJS ). The site is live here http://www.nationwidecoils.com/ if that helps.

**Comment by cjeria** [comment]
I'm having a similar issue. I'm loading the js file in the <head> of my site.

![image](https://cloud.githubusercontent.com/assets/597770/7657893/9fe3f0ba-faea-11e4-8bff-f885aa9e5bca.png)

**Comment by desandro** [comment]
If you can provide a **reduced test case**, I'd love to take a look.

Most likely, Flickity is initialized multiple times on the same element.

**Comment by cjeria** [comment]
I wasn't able to replicate the problem in my reduced test case, which works as expected. It only seems to be happening in my dev environment. I'm using a static site generator Middlemanapp https://github.com/middleman/middleman

Here's my site https://github.com/cjeria/visualty, I'll add you as a collaborator. Thx!

**Comment by cjeria** [comment]
Solved this issue. If I understand this correctly, the JS was being loaded twice, once from the node_modules directory (I used `npm install flickity`) and from the head of my template. I simply deleted the one from the head and it works at it should. The CSS file however, needs to be called out in the head. Would be nice if it would load the same way as the JS file does with node_modules. Not sure how to configure that.
